setuper/data_collection.py

```diff
@@ -34,81 +31,158 @@ def setup_instances(account_name, iaso_client):
+                        ]
+                    ),
+                    "engagements_rh": random.choice(
+                        [
+                            "volontaires securisation_des_mobilisateurs autre_engagement_rh",
+                            "securisation_des_mobilisateurs autre_engagement_rh",
+                            "volontaires autre_engagement_rh",
+                            "volontaires securisation_des_mobilisateurs",
+                        ]
+                    ),
+                    "engagements_materiel": random.choice(
+                        [
+                            "logistique_de_poste_de_vaccination__tent moyen_de_d_placement__moto__v_lo__pirogu carburant",
+                            "rafra_chissement__eau__nourriture__banan autre_engagement_materiel",
+                            "logistique_de_poste_de_vaccination__tent moyen_de_d_placement__moto__v_lo__pirogu carburant rafra_chissement__eau__nourriture__banan autre_engagement_materiel",
+                        ]
+                    ),
+                    "lieu_rdv": random.choice(["chefferies", "lieux_de_culte", "march_s", "ecoles", "autres"]),
+                    "date_rdv": date_rdv.date(),
+                },
+                "microplan": {
+                    "Population_estimee_du_village": estimated_pop_village,
+                    "strategie": random.choice(["1 2 3 4", "1 2", "3 4", "1 3", "1 4", "2 3", "2 4"]),
+                },
+                "meta": {"instanceID": "uuid:" + the_uuid},
+            }
 
-        ## mobile code
-        # https://github.com/BLSQ/iaso-mobile-app/blob/develop/odk-collect/src/main/java/org/odk/collect/android/tasks/InstanceServerUploaderTask.java#L88
-        # https://github.com/BLSQ/iaso-mobile-app/blob/develop/odk-collect/src/main/java/org/odk/collect/android/upload/InstanceServerUploader.java#L70
-        # https://github.com/BLSQ/iaso-mobile-app/blob/develop/collect_app/src/main/java/com/bluesquare/iaso/usecase/SyncInstances.kt
-        if count % 5 == 0:
-            print("\t%d submissions done" % count)
+            # see hat/sync/views.py
+            image_number = (count % 3) + 1
+            with open(f"./data/fosa{image_number}.jpeg", "rb") as fp_image:
+                iaso_client.post(
+                    "/sync/form_upload/",
+                    files={
+                        "xml_submission_file": (
+                            local_path,
+                            submission2xml(
+                                instance_json, form_version_id=form_version["version_id"], form_id="SAMPLE_FORM_new5"
+                            ),
+                        ),
+                        "imgUrl": fp_image,
+                    },
+                )
+
+            count = count + 1
+
+            ## mobile code
+            # https://github.com/BLSQ/iaso-mobile-app/blob/develop/odk-collect/src/main/java/org/odk/collect/android/tasks/InstanceServerUploaderTask.java#L88
```

Comment:
```suggestion
            # https://github.com/BLSQ/iaso-mobile-app/blob/0c6d821056e41d5beb6f745ea807451b07eb35f2/odk-collect/src/main/java/org/odk/collect/android/tasks/InstanceServerUploaderTask.java#L88
```

`develop` will change over time. You may want to put link to a permanent change.